proto/environment/command.proto

```diff
@@ -63,3 +63,28 @@ message EnableProjectCommand {}
 message DisableProjectCommand {}
 
 message ConvertTrialProjectCommand {}
+
+message CreateOrganizationCommand {
+  string name = 1;
+  string url_code = 2;
+  string description = 3;  // optional
+  string is_trial = 4;
```

Comment:
Please use boolean instead.

Reply:
@cre8ivejp 
Thank you, nice catch!! 🙏 🙏